R/eval-relocate.R

```diff
@@ -83,6 +89,7 @@ eval_relocate <- function(expr,
     name_spec = name_spec,
     allow_rename = allow_rename,
     allow_empty = allow_empty,
+    allow_predicates = allow_predicates,
     type = "relocate",
     error_call = error_call
   )
```

Comment:
I think the `eval_select()` calls for `.before` and `.after` (further below) need to know about this `allow_predicates` argument somehow.

Assuming `data <- tidyselect_data_proxy(data)` returns an R data frame if it was original a lazy df, then the `eval_select()` call for `before` and `after` would look like they _do_ support predicates, but they really shouldn't.